apps/package.json

```diff
@@ -27,7 +27,10 @@
   },
   "browserify-global-shim": {
     "react": "React",
-    "react-dom": "ReactDOM"
+    "react-dom": "ReactDOM",
+    "johnny-five": "JohnnyFive",
+    "playground-io": "PlaygroundIO",
+    "chrome-serialport": "ChromeSerialport"
```

Comment:
Are these actually used in apps at all yet? (I dont see any other changes to apps in this PR)